spec/mailtrap/projects_api_spec.rb

```diff
@@ -0,0 +1,198 @@
+# frozen_string_literal: true
+
+RSpec.describe Mailtrap::ProjectsAPI, :vcr do
+  subject(:project) { described_class.new(account_id, client) }
+
+  let(:account_id) { ENV.fetch('MAILTRAP_ACCOUNT_ID', 1_111_111) }
+  let(:client) { Mailtrap::Client.new(api_key: ENV.fetch('MAILTRAP_API_KEY', 'local-api-key')) }
+
+  describe '#list' do
+    subject(:list) { project.list }
+
+    it 'maps response data to Project objects' do
+      expect(list).to all(be_a(Mailtrap::Project))
+      expect(list.first).to have_attributes(
+        id: be_a(Integer),
+        name: be_a(String)
+      )
```

Comment:
`all(be_a(Mailtrap::Project))` already ensures specific type. What is the intention for asserting specific attributes on first item?